### PR TITLE
Add core WG notes for 2022-10-07

### PR DESCRIPTION
**Pull request by jrvanwhy**
[Rendered](https://github.com/jrvanwhy/tock/blob/core-notes-2022-10-07/doc/wg/core/notes/core-notes-2022-10-07.md)

### CODE REVIEW

File: doc/wg/core/notes/core-notes-2022-10-07.md [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+   handling.
+ * Pat: It's a longstanding pain point so I'm onboard with getting this to
+   happen.
+
+## Component update PRs
+ * Pat: On the subject of large mechanical changes, Brad has a small army of
+   component PRs. I reviewed a bunch, tagged them last night. We'll probably
+   merge them now, unless someone has a reason not to, we should probably get
+   those in.
+ * Phil: I `bors r=`-ed the basic one.
```
Comment (ppannuto) [inline doc/wg/core/notes/core-notes-2022-10-07.md:363]:
```suggestion
 * Phil: I `bors r+`-ed the basic one.
```

**Review by ppannuto — APPROVED** [review]
_[33 second silence]_

😆

File: doc/wg/core/notes/core-notes-2022-10-07.md (hunk 1 above)
Comment (jrvanwhy) [reply doc/wg/core/notes/core-notes-2022-10-07.md]: Oops, somehow missed this for 2 weeks. Done!

**Review by ppannuto — APPROVED** [review]

**Pull request merged**